.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19
+      - name: Verify dependencies
+        run: go mod verify
+      - name: Build
+        run: go build -v ./...
+      - name: Run tests
+        run: go test -race -vet=off ./...
```

Comment:
What is the deal with `-vet=off`?

Reply:
As part of building a test binary, go test runs go vet on the package
and its test source files to identify significant problems. If go vet
finds any problems, go test reports those and does not run the test
binary. https://pkg.go.dev/cmd/go/internal/test

I can remove this param

Reply:
:+1: for removing. Having some basic validation is helpful.